CMakeLists.txt

```diff
@@ -137,36 +143,27 @@ if(NOT WIN32)
 endif()
 string(REPLACE ";" " " CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
 
-# Use RUNPATH instead of RPATH for all shared libs and executables on Linux
-if(IS_LINUX)
-  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,--enable-new-dtags")
-  set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,--enable-new-dtags")
-endif()
-
-#==============================================================================
-# Tests
-#==============================================================================
+#------------------------------------------------------------------------------
```

Comment:
Leaving compiler configuration as is for now!! This will change in the second phase of cmake clean up.